starport/services/network/networkchain/tunneled_peers.go

```diff
@@ -0,0 +1,65 @@
+package networkchain
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+const TunneledPeersFile = "tunneled_peers.json"
+
+type TunneledPeerConfig struct {
+	TunneledPeers []TunneledPeer `json:"tunneled_peers"`
```

Comment:
```suggestion
	TunneledPeers []TunneledPeer `json:"tunneledPeers"`
```

Reply:
I don't understand. As I see all field names in config files are presented in snake case.

Reply:
that's right, let's keep underscores.